gateway/src/apicast/policy/jwt_claim_check/jwt_claim_check.lua

```diff
@@ -0,0 +1,55 @@
+local policy = require('apicast.policy')
+local _M = policy.new('JWT check policy')
+
+local Operation = require('apicast.conditions.operation')
+-- local Condition = require('apicast.conditions.condition')
+
+local new = _M.new
+
+local default_error_message = "Request block due JWT claim policy"
+
+local function deny_request(error_msg)
+  ngx.status = ngx.HTTP_FORBIDDEN
+  ngx.say(error_msg)
+  ngx.exit(ngx.status)
+end
+
+
+local function verdict(check_type, status)
+  -- Default whitelist mode, if not valid using whitelist
+  if check_type == "blacklist" then
+    return not status
+  end
+
+  return status
+end
+
+function _M.new(config)
+  local self = new(config)
+  self.error_message = config.error_message or default_error_message
+
+  self.conditions = {}
+  for _,condition in ipairs(config.operations) do
+    if condition.match_type == "plain" then
+      -- Due to this need to be fetched from the JWT claim, render  the match
+      -- as liquid to be able to fetch the info from the JWT_claim
+      condition.match = "{{"..condition.match.."}}"
+    end
+    table.insert(self.conditions, {
+      op = Operation.new(condition.match, "liquid", condition.op, condition.value, 'plain'),
+      check_type = condition.check_type})
+  end
+  return self
+end
+
+
+function _M:access(context)
+  for _,operation in pairs(self.conditions) do
```

Comment:
I think it would be good to support 'conditions' like in other policies like the routing one.
That would be more consistent, and also would allow us to combine operations with `and` and `or`.